src/ApiClient.js

```diff
@@ -60,6 +60,9 @@ class ApiClient {
     if (param instanceof Date) {
       return param.toJSON();
     }
+    if (param instanceof Object) {
+      return param;
```

Comment:
object query params will be encoded correctly by the library so they must be returned as is. Otherwise `toString` is called on them and we get `"Object"`